### ISSUE TITLE
How does IFC.js future

### ISSUE THREAD

**Issue by steven11329** [IFCjs/web-ifc#511]
Hi IFCjs Team.
I've noticed that the website [IFC.js document](https://ifcjs.github.io/info/docs/Guide/Getting%20started/]) has been shut down. 
Would you have any plans to reopen it in the future, or should we consider using a different library instead of IFC.js?

**Comment by agviegas** [comment]
Hi @steven11329,
We have moved the documentation [here](https://platform.thatopen.com/documentation) and the community [here](https://people.thatopen.com/). We are growing a lot and made a [launch](https://www.youtube.com/watch?v=c9aj5CXOQs0) announcing where we are and where we are going. Bottom line is that we have now built a company to accelerate development, and all present and future libraries will remain MIT. Let us know if you have any questions.
